packages/pixeloven-webpack/config/src/types.ts

```diff
@@ -13,6 +13,17 @@ export interface Config {
     };
 }
 
+export interface CompilerConfig {
```

Comment:
@ductiletoaster let me know if you have a better name here. It's getting hard to follow what is what because there's multiple objects with similar names/structure.

Reply:
I don't have a good suggestion right now but I totally agree it is hard to follow.